2-Authorization-I/1-call-graph/SPA/src/components/NavigationBar.jsx

```diff
@@ -1,44 +1,100 @@
+                        Mails
+                    </Nav.Link>
+                    <Nav.Link as={Button} className="navbarButton" href="/tenant">
+                        Tenant
+                    </Nav.Link>
+                    <div className="collapse navbar-collapse justify-content-end">
+                        <DropdownButton
+                            variant="warning"
+                            drop="start"
+                            title={activeAccount ? activeAccount.name : 'Unknown'}
+                        >
+                            <Dropdown.Item as="button" onClick={handleSwitchAccount}>
+                                Switch account
+                            </Dropdown.Item>
+                            <Dropdown.Item
+                                as="button"
+                                onClick={() =>
+                                    instance.logoutPopup({
+                                        postLogoutRedirectUri: msalConfig.postLogoutRedirectUri,
+                                    })
+                                }
+                            >
+                                Sign out using Popup
+                            </Dropdown.Item>
+                            <Dropdown.Item
+                                as="button"
+                                onClick={() =>
+                                    instance.logoutRedirect({
+                                        postLogoutRedirectUri: msalConfig.postLogoutRedirectUri,
+                                    })
+                                }
+                            >
+                                Sign out using Redirect
+                            </Dropdown.Item>
+                        </DropdownButton>
+                    </div>
                 </AuthenticatedTemplate>
                 <UnauthenticatedTemplate>
-                    <DropdownButton variant="secondary" className="ml-auto" drop="left" title="Sign In">
-                        <Dropdown.Item as="button" onClick={handleLogin}>Sign in using Popup</Dropdown.Item>
-                        <Dropdown.Item as="button" onClick={() => instance.loginRedirect(loginRequest)}>Sign in using Redirect</Dropdown.Item>
-                    </DropdownButton>
+                    <div className="collapse navbar-collapse justify-content-end">
+                        <DropdownButton
+                            variant="secondary"
+                            className="justify-content-end ml-auto"
+                            title="Sign In"
+                            drop="start"
+                        >
+                            <Dropdown.Item as="button" onClick={handleLogin}>
+                                Sign in using Popup
+                            </Dropdown.Item>
+                            <Dropdown.Item as="button" onClick={() => instance.loginRedirect(loginRequest)}>
```

Comment:
```suggestion
                            <Dropdown.Item as="button" onClick={() => instance.loginRedirect(loginRequest)}>
```

let's preserve the same pattern here i.e. use a `handleLoginRedirect` method

Reply:
Done